modules/sticky-table-headers/main/index.coffee

```diff
@@ -12,20 +12,19 @@ updateScrollIndicators = (wrapper, top, right, bottom, left) ->
   bottom.style('display', if canScrollDown then 'block' else '')
   left.style('display', if canScrollLeft then 'block' else '')
 
-updateHeaderPositions = (container) ->
+updateHeaderPositions = (container, wrapperNode) ->
   # Set the relative positions of the two headers.
   # This method reduces flickering when scrolling on mobile devices.
-  node = getChildren(container, '.hx-sticky-table-wrapper')[0]
```

Comment:
Replacing this with `shallowSelect` made some of the data table tests fail (as it couldn't find `hx-sticky-table-wrapper) so I passed it into the function instead - I think it was because `PhantomJS` doesn't 100% correctly with detached nodes (as it worked fine in the browser)